Change order and organization of New Features in WCAG 2.2

yatil opened the pull request:
This change makes the list of success criteria look similar to the same list in WCAG 2.1. This is done to ensure that it’s easy to see what new SCs are there and what their level and SC number is.

- Fixes w3c/wcag#1807

Signed-off-by: Eric Eggert <mail@yatil.net>


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/yatil/wcag/pull/2667.html" title="Last updated on Sep 6, 2022, 7:17 PM UTC (de6a6a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/2667/17f8477...yatil:de6a6a3.html" title="Last updated on Sep 6, 2022, 7:17 PM UTC (de6a6a3)">Diff</a>

patrickhlauke reviewed (approved):
agree that's more useful

yatil commented:
I waive all my IPR that could be associated with this pull request.

awkawk commented on guidelines/index.html:
```diff
@@ -91,15 +91,15 @@ <h4>New Features in WCAG 2.2</h4>
+						<li>3.3.7 <a href="#accessible-authentication">Accessible Authentication</a> (AA)</li>
+						<li>3.3.8 <a href="#accessible-authentication-no-exception">Accessible Authentication (No Exception)</a> (AAA)</li>
+						<li>3.3.9 <a href="#redundant-entry">Redundant Entry</a> (A)</li>
+						<li>3.5.7 <a href="#dragging-movements">Dragging Movements</a> (AA)</li>
```
This is 2.5.7 so should move up the list

awkawk commented on guidelines/index.html:
```diff
@@ -91,15 +91,15 @@ <h4>New Features in WCAG 2.2</h4>
+						<li>3.3.8 <a href="#accessible-authentication-no-exception">Accessible Authentication (No Exception)</a> (AAA)</li>
+						<li>3.3.9 <a href="#redundant-entry">Redundant Entry</a> (A)</li>
+						<li>3.5.7 <a href="#dragging-movements">Dragging Movements</a> (AA)</li>
+						<li>3.5.8 <a href="#target-size-minimum">Target Size (Minimum)</a> (AA)</li>
```
This is 2.5.8 so should move up the list

yatil replied:
Fixed the PR.

bruce-usab commented:
Can this PR include the link to https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/ per SLH request?  Or should that be its own PR?

yatil commented:
> Can this PR include [what's new in 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/) link per [SLH request](https://github.com/w3c/wcag/issues/1807#issuecomment-1204475693)?  Or should that be its own PR?

It does not, as that was a [separate issue in my original issue](https://github.com/w3c/wcag/issues/1807#issuecomment-1238356727) without any actionable advice on how or what to link and I did not want to include any editorial text. I’m only interested in fixing the list at this point.

yatil commented:
What is holding up this PR from being merged?

yatil commented:
Heeeeellloooooooo? 😅

yatil commented:
Is there anybody here? _echoing_ here here here here…

awkawk commented:
@yatil This is certainly editorial and I'm sure that it will be addressed in the comment period. :)

yatil commented:
It would just be nice to have some recognition that this is looked at. Ignoring contributions from volunteers is just not encouraging, if not demoralizing. Plus, I have been told that some contributions that have been in the repository for years are “too late to be included” now, as if it is a contributor’s responsibility that the WG addresses issues. (Maybe it is, and I misunderstood.) Hence my pinging this semi-regularly.

yatil commented:
Bump.

yatil commented:
One-month-iversary of this Pull Request! 🥳🥳🥳

hidde commented:
This looks like a sensible change that adds consistency and breaks nothing. Why can't it be merged?

yatil commented:
Bump.

yatil commented:
Bump.

MelSumner reviewed (approved):
💯 nice edit, thank you for doing this work!

yatil commented:
Bump again.

yatil commented:
Just one week until the 2-monthiversary of this PR! I’m going to buy some cake!

alastc commented:
There was a lot of spacing removals in the PR (probably an editor/IDE automatic thing?), so I copied it across to #2780 and merged.

Thanks for the effort on this @yatil. I had to take a bit of a break from github issues after CR, but we're motoring through now...
